packages/web-react/src/components/ModalComposed/README.md

```diff
@@ -0,0 +1,84 @@
+# Modal
+
+```jsx static
+import {
+  ModalComposed,
+  ModalComposedDialog,
+  ModalComposedBody,
+  ModalComposedHeader,
+  ModalComposedFooter,
+} from '@lmc-eu/spirit-web-react/components';
+```
+
+```jsx
+todo;
+```
+
+## Modal
+
+**Available props**
+
+| Name                 | Type                                           | Default | Required | Description                                    |
+| -------------------- | ---------------------------------------------- | ------- | -------- | ---------------------------------------------- |
+| `id`                 | `string`                                       | -       | yes      | Open state                                     |
+| `isOpen`             | `boolean`                                      | `false` | yes      | Open state                                     |
+| `onClose`            | `(event: ClickEvent or KeyboardEvent) => void` | -       | yes      | Callback on dialog closed                      |
+| `children`           | `ReactNode`                                    | -       | no       | Children node                                  |
+| `UNSAFE_className`   | `string`                                       | -       | no       | Modal custom class name                        |
+| `UNSAFE_style`       | `CSSProperties`                                | -       | no       | Modal custom style                             |
+| `isExpandedOnMobile` | `boolean`                                      | -       | no       | Modal shrinks to fit the height of its content |
```

Comment:
After the separation of `ModalDialog`, this option needs to be moved there.

Reply:
done